src/Core/src/Platform/Android/SemanticExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Android.Text;
+using Android.Views;
+using Android.Views.Accessibility;
+
+namespace Microsoft.Maui
+{
+	public static partial class SemanticExtensions
+	{
+		public static void SetSemanticFocus(IFrameworkElement element)
+		{
+			var view = element.GetViewForAccessibility();
+
+			view?.SendAccessibilityEvent(EventTypes.ViewFocused);
+		}
+
+		internal static View? GetViewForAccessibility(this IFrameworkElement element)
```

Comment:
This shouldn't be needed anymore for MAUI
this was specific to how forms and renderers worked